services/api-server/src/simcore_service_api_server/models/domain/projects.py

```diff
@@ -0,0 +1,35 @@
+# from datetime import datetime
+# from typing import Optional
+
+from models_library.projects import AccessRights, Node, Project, StudyUI
+from models_library.projects_nodes import InputTypes, OutputTypes
+from models_library.projects_nodes_io import SimCoreFileLink
+
+from ...utils.serialization import json_dumps, json_loads
+
+
+class NewProjectIn(Project):
+    """ Web-server API model in body for create_project """
+
+    # - uuid
+    # - name
+    # - description
+    # - prjOwner
+    # - accessRights
+    # - creationDate
+    # - lastChangeDate
+    # - thumbnail
+    # - workbench
+    class Config:
+        json_loads = json_loads
+        json_dumps = json_dumps
```

Comment:
is that using orjson? we might have things to check with the spacings. since orjson is extremly compacting

Reply:
well if I'm remembering this correctly:
```{"myjson": "my blahblah"}```
1. json lib would serialize to ```{\"myjson\": \"my blahblah\"}```
2. orjson lib would serialize to ```{\"myjson\":\"my blahblah\"}```

note the spacing difference around the ```:``` character

Reply:
but if you refer to the hash from the director-v2, it is not affected by whatever is in the api-server. unless you try to compute the run_hash of comp_tasks. which you are not